Library/Homebrew/cmd/test-bot.rb

```diff
@@ -209,19 +209,16 @@ def initialize(argument, options={})
       @added_formulae = []
       @modified_formula = []
       @steps = []
-      @tap = options[:tap]
+      @tap = options.fetch(:tap,  CoreFormulaRepository.instance)
       @repository = @tap.path
-      @skip_homebrew = options[:skip_homebrew]
+      @skip_homebrew = options.fetch(:skip_homebrew, false)
```

Comment:
I'd personally favour `!!options[:skip_homebrew]` if you want it to be a boolean.